src/sentry/api/endpoints/organization_member/details.py

```diff
@@ -49,6 +47,47 @@
 ERR_EXPIRED = "You cannot resend an expired invitation without regenerating the token."
 ERR_RATE_LIMITED = "You are being rate limited for too many invitations."
 
+_team_roles_description = """
+Configures the team role of the member. The two roles are:
+- `contributor`: Can view and act on issues. Depending on organization settings, they can also add team members.
+- `admin`: Has full management access to their team's membership and projects.
+```json
+{
+    "teamRoles": [
+        {
+            "teamSlug": "ancient-gabelers",
+            "role": "admin"
+        },
+        {
+            "teamSlug": "powerful-abolitionist",
+            "role": "contributor"
+        }
+    ]
+}
+```
+"""
+
+# Required to explictly define roles w/ descriptions because OrganizationMemberSerializer
+# has the wrong descriptions, includes deprecated admin, and excludes billing
+_role_choices = [
+    ("billing", "Can manage payment and compliance details."),
```

Comment:
the billing role doesn't exist in sentry. is it a concern if we mention it here?

Reply:
I don't think this is a concern as the API takes this in, and we allow people to switch on the UI

Reply:
I agree. This is basically how I'm publishing spike protection too. The whole thing is in getsentry but if you have the right permissions and you can call it why not have it documented.